tests/app/celery/test_tasks.py

```diff
@@ -654,7 +649,7 @@ def test_send_email_should_go_to_retry_queue_if_database_errors(sample_email_tem
     assert 'No row was found for one' in str(e.value)
 
 
-def test_process_email_job_should_use_reply_to_email_if_present(sample_email_job, mocker, mock_celery_remove_job):
+def test_process_email_should_not_use_reply_to_email(sample_email_job, mocker, mock_celery_remove_job):
```

Comment:
yeah i removed it in next PR - had it in this one to sanity check that my two-step change still accepts the old parameters